package.json

```diff
@@ -34,33 +34,7 @@
   "activationEvents": [
     "onDebugInitialConfigurations",
     "onDebug",
-    "onView:stripeEventsView",
-    "onView:stripeLogsView",
-    "onView:stripeQuickLinksView",
-    "onView:stripeHelpView",
-    "onCommand:stripe.createStripeSample",
-    "onCommand:stripe.createWebhookEndpoint",
-    "onCommand:stripe.login",
-    "onCommand:stripe.openCLI",
-    "onCommand:stripe.openWebhooksListen",
-    "onCommand:stripe.openDashboardApikeys",
-    "onCommand:stripe.openDashboardLogs",
-    "onCommand:stripe.openDashboardEvents",
-    "onCommand:stripe.openDashboardWebhooks",
-    "onCommand:stripe.openEventDetails",
-    "onCommand:stripe.openTriggerEvent",
     "onCommand:stripe.openCreateCustomizedEvent",
-    "onCommand:stripe.openTriggerCustomizedEvent",
-    "onCommand:stripe.openReportIssue",
-    "onCommand:stripe.openDocs",
-    "onCommand:stripe.openSurvey",
-    "onCommand:stripe.openSamples",
-    "onCommand:stripe.startLogsStreaming",
-    "onCommand:stripe.stopLogsStreaming",
-    "onCommand:stripe.clearRecentLogs",
-    "onCommand:stripe.clearRecentEvents",
-    "onCommand:stripe.startEventsStreaming",
-    "onCommand:stripe.stopEventsStreaming",
```

Comment:
Do we need to get rid of these activation events? I suppose the `onView` ones are noop because the extension would've already been active when you view them but the `onCommand` ones seem important.

Reply:
(This is @matv) - in newer version of VSCode, the linter suggested I remove these.

https://code.visualstudio.com/api/references/activation-events#onCommand

> Note: Beginning with VS Code 1.74.0, commands contributed by your extension do not require a corresponding onCommand activation event declaration for your extension to be activated.